developers/blobstream-x-deploy.md

```diff
@@ -0,0 +1,63 @@
+# Non-canonical Blobstream X deployments
+
+If you want to deploy Blobstream X to a new chain, where a
+canonical Succinct Gateway contract does not exist, you
+need to do the following.
+
+1.  Deploy a new `SuccinctGateway` contract to the new chain.
```

Comment:
@jcstein these steps will be moved to our documentation since I think they can be generically helpful for anyone who is trying to use Succinct, not just celestia users, so I'll ping you with the links once done.

Reply:
sounds good! i'll leave this unresolved for tracking purposes